operator/Makefile

```
@@ -154,6 +151,38 @@ reporting: $(REPORTING)
+
+.PHONY: print-k8s-test-versions-github-output
+print-k8s-test-versions-github-output: ## Print Kubernetes test versions in GitHub Actions output format.
+	@printf 'envtest-k8s-version=%s\n' '$(ENVTEST_K8S_VERSION)'
+	@printf 'kind-version=%s\n' '$(KIND_VERSION)'
+	@printf 'kind-node-image-version=%s\n' '$(KIND_NODE_IMAGE_VERSION)'
+	@printf 'kind-binary-version=%s\n' '$(KIND_BINARY_VERSION)'
+	@printf 'ci-kind-node-image-versions-json=%s\n' '$(CI_KIND_NODE_IMAGE_VERSIONS_JSON)'
+	@printf 'ci-primary-kind-node-image-version=%s\n' '$(CI_PRIMARY_KIND_NODE_IMAGE_VERSION)'
+
+.PHONY: validate-kind-node-image
+validate-kind-node-image: ## Check that the configured kindest/node image tag exists before creating a cluster.
```

checkmake (0.3.2)

[warning] Target body for "validate-kind-node-image" exceeds allowed length of 5 lines (12).

(maxbodylength)